README.md

```diff
@@ -0,0 +1,10 @@
+# selenium-ta-project
+
+To run tests execute following command:
+
+```
+mvn -Dbrowser=chrome test
+```
+browser parameter can be changed, available options at this moment:
+- chrome
+- firefox
```

Comment:
please add info how to generate the Report

Reply:
report wip